core/src/main/java/jenkins/telemetry/Telemetry.java

```diff
@@ -141,6 +141,22 @@ public static boolean isDisabled() {
         return jenkins == null || !jenkins.isUsageStatisticsCollected();
     }
 
+    /**
+     * Returns true if and only if we are in the time period during which this is supposed to collect data.
+     * @return true iff we're in the time period during which this is supposed to collect data
```

Comment:
can I get you to use `IFF` instead of `iff`? it looks too close to a typo as is.

Reply:
IFF looks like an acronym.

Identification Friend or Foe. Which are you? 😆

Reply:
IFF is a mathematical abbreviation for "if and only if". I don't see any advantage to using the abbreviation in javadocs.